nixos/modules/services/web-apps/stirling-pdf.nix

```diff
@@ -110,7 +130,13 @@ in
         RestrictRealtime = true;
         SystemCallArchitectures = "native";
         SystemCallFilter = [
```

Comment:
Can you explain more about the change in the `SystemCallFilter`? Specifying `SystemCallFilter` multiple times is different from using space-separated option, and it means that the following occurrences will add or delete the listed system calls from the previous set of the filtered system calls.

(See more at https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#System%20Call%20Filtering)

Reply:
Indeed there is a slight difference. I think in this precise case those are equivalent but I do agree I should revert it.